modules/core/src/shaderlib/misc/geometry.js

```diff
@@ -18,13 +18,20 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-export default `\
-highp float random(vec2 co) {
-  highp float a = 12.9898;
-  highp float b = 78.233;
-  highp float c = 43758.5453;
-  highp float dt= dot(co.xy ,vec2(a,b));
-  highp float sn= mod(dt,3.14);
-  return fract(sin(sn) / c) - .5;
-}
+const vs = `
+struct VertexGeometry {
+  vec3 worldPosition;
+  vec3 worldPositionAlt;
+  vec4 position;
```

Comment:
Would suggest ordering members in decreasing order of size. It's a convention that tends to pack things tightly (in C and std140 at least, not 100% sure for GLSL in general).